cdap-standalone/pom.xml

```diff
@@ -633,6 +633,14 @@
                         <include>*.sdk.jar</include>
                       </includes>
                     </resource>
+                    <!-- copy bootstrap file -->
+                    <resource>
+                      <directory>${project.parent.basedir}/cdap-common/src/main/resources/bootstrap</directory>
+                      <targetPath>bootstrap</targetPath>
+                      <includes>
+                        <include>sandbox.json</include>
+                      </includes>
+                    </resource>
```

Comment:
why is this required ?

Reply:
this is needed for packaging purposes, to copy the bootstrap file to the right place. If you're asking why there's a separate file for distributed and sandbox, it's because distributed doesn't auto-start dataprep, but sandbox does. Will change the bootstrap files accordingly once the start program type is implemented